c++/src/kj/compat/http-test.c++

```diff
@@ -1909,31 +1903,6 @@ KJ_TEST("HttpServer WebSocket handshake") {
   listenTask.wait(io.waitScope);
 }
 
-KJ_TEST("HttpServer WebSocket handshake detached") {
```

Comment:
This test was testing that you could extend a WebSocket's lifetime beyond the request handler promise, which was a dubious feature that I never relied on. That feature is now removed.